src/pytest_mock_resources/container/redis.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+
+from pytest_mock_resources.compat import redis
+from pytest_mock_resources.container import (
+    ContainerCheckFailed,
+    get_container_fn,
+    get_docker_host,
+)
+
+# XXX: To become overwritable via pytest config.
+config = {
+    "port": 6379,
+    "image": "redis:5.0.7",
+}
+
+
+def check_redis_fn():
+    try:
+        client = redis.Redis(host=get_docker_host(), port=config["port"])
+        client.ping()
+    except redis.ConnectionError:
+        raise ContainerCheckFailed(
+            "Unable to connect to a presumed Redis test container via given config: {}".format(
+                config
+            )
+        )
+
+
+_redis_container = pytest.fixture("session")(
```

Comment:
I feel like we should start with this as a non-session fixture, so that it's guaranteed to be empty on  each invocation (and add a test which asserts that there are no keys written, which we'd expect to be run after other tests).

Then once we have mechanisms for ensuring inter-test data-cleanliness, we can make this more efficient by making it a session fixture